resources/packs/core/core.lr

```diff
@@ -1,10 +1,10 @@
 option go_package = "go.mondoo.com/cnquery/resources/packs/core"
 
-// Provide contextual info about Mondoo Client and environment
+// Provide contextual info about cnquery and environment
```

Comment:
This requires a re-generate of the docs files. `make cnquery/generate` will do this for you. Then, also the tests should pass.

Reply:
Updated and rebased